surefire-api/src/test/java/org/apache/maven/surefire/api/report/LegacyPojoStackTraceWriterTest.java

```diff
@@ -64,14 +65,18 @@ public void testWriteTrimmedTraceToString()
         MockThrowable t = new MockThrowable( stackTrace );
         LegacyPojoStackTraceWriter w = new LegacyPojoStackTraceWriter( "TestSurefire3", "testQuote", t );
         String out = w.writeTrimmedTraceToString();
-        String expected = "junit.framework.AssertionFailedError: blah\n" + "    at junit.framework.Assert.fail(Assert.java:47)\n" + "    at TestSurefire3.testQuote(TestSurefire3.java:23)\n";
+        String expected = "junit.framework.AssertionFailedError: blah\n"
```

Comment:
I'm confused. This seems to add an extra line to the output and not remove anything. Is there a test that shows these changes making stack traces simpler?